.github/docker/Dockerfile.debian

```diff
@@ -1,3 +1,3 @@
-FROM debian:bullseye-20231120
+FROM debian:bookworm-20231120
```

Comment:
I know we tried this before and some things failed but now there have been a few minor releases and I'd expect newer versions to be compatible.